src/ui/public/vis_maps/geohash_layer.js

```diff
@@ -0,0 +1,574 @@
+import KibanaMapLayer from './kibana_map_layer';
```

Comment:
seems you have multiple class definitions in this file ? ScaledCircles, ShadedCircles .... this was already split out before right ? any reason to bring it all together in this huge file ?